ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpSourceSpec.scala

```diff
@@ -3,10 +3,14 @@
  */
 package akka.stream.alpakka.ftp
 
+import java.nio.file.attribute.PosixFilePermission
```

Comment:
So far, I've been putting the java imports at the end of the import list. Just by convention.